File: services/GroupService/src/main/java/com/se1853_jv/dto/response/PaperResponse.java
package com.se1853_jv.dto.response;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.List;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class PaperResponse {
    private String id;
    private String dataUrl;
    private List<String> keywords;
    private String title;
    private String authors;
    private String journal;
    private Integer publicationYear;
    private String doi;
    private String description;
}

File: services/paper-service/src/main/kotlin/com/se1853_jv/dto/response/PaperResponse.kt
package com.se1853_jv.dto.response

data class PaperResponse(
    val id: String,
    val dataUrl: String,
    val keywords: List<String>? = null,
    val title: String,
    val authors: String,
    val journal: String,
    val publicationYear: Int,
    val doi: String,
    val description: String?,
)
